url/resources/urltestdata.json

```diff
@@ -7217,4 +7344,4 @@
     "search": "",
     "username": "user"
   }
-]
+]
```

Comment:
Please restore the newline at the end.

Reply:
Ah, I didn't see that.

Reply:
Just so I know how this works, does each such commit send out another approval request to each of the reviewers?

Reply:
I think that depends on how you configure your GitHub notifications, but maybe? I usually wait for some kind of comment before taking another look.